kibbeh/src/modules/layouts/ElectronHeader.tsx

```diff
@@ -49,7 +49,7 @@ function WinHeader() {
           }}
         />
         <WinButton
-          icon={<WinMinimizeIcon width={10} height={10} />}
+          icon={<WinMinimizeIcon width={10} height={1} />}
```

Comment:
i don't see how changing the height of this will make a difference, also won't this make the icon nearly invisible

Reply:
Before: 
![image](https://user-images.githubusercontent.com/50546858/116857649-756bb280-abfd-11eb-9b49-5300291703b0.png)

After:
![image](https://user-images.githubusercontent.com/50546858/116857695-861c2880-abfd-11eb-9493-dcb8d8f45f90.png)

Reply:
i see 0 difference

Reply:
there is no need for this, please remove this

Reply:
How do you not see any difference?

Reply:
im blind